src/locals/hu.json

```diff
@@ -0,0 +1,145 @@
+    "catastrophizing_explanation": "Ha a bolhából is elefántots csinálunk, akkor katasztofizálunk. Ha egy kis hibát ejtettél a munkahelyeden, még nem biztos, hogy csak a miatt kirúgnak.",
+    "catastrophizing_thought": "Ideges vagyok, biztos szívrohamom van.",
+    "emotional_reasoning": "Érzelmi érvelés",
+    "emotional_reasoning_explanation_1": "\"Érzem, tehát biztos igaz is.\" Ha jogosnak érzed a \"veszélyt\" azért, mert te félsz tőle, akkor elképzelhető, hogy érzelmi érvelést alkalmazol. A dolgok nem azért veszélyesek, mert mi félünk tőle vagy azt gondoljuk, hogy rossz.",
+    "emotional_reasoning_explanation_2": "Ezt sokszor nehéz érzékelni. Nehéz összeszedned magad annyira, hogy logikusan tudj gondolkozni az érzelmeidről is.",
+    "emotional_reasoning_thought": "Bűnösnek érzem magam, ezért biztos, hogy rosz fát tettem a tűzre.",
+    "fortune_telling": "Jövendőmondás",
+    "fortune_telling_explanation": "Amikor ki akarjuk találni a jövőt. Sokszor túlbecsüljük a jövendőmondó képességünket. Sokszor erős előfeltételezésekkel élünk és csak az azt alátámasztó bizonyítékokat vesszük figyelembe.",
+    "fortune_telling_thought": "Le fog zuhanni a repülő, amin ülök.",
+    "labeling": "Címkézés",
+    "labeling_explanation": "Ha egy jellegzetes tulajdonsággal azonosítunk egy embert, akkor címkézünk. Ha valaki bevágott eléng, attól még nem biztos, hogy ő \"*eggfej,\" lehet, hogy csak siet. Ez ránk is vonatkozik; csak azért, mert hibázunk még nem vagyunk \"hibásak.\"",    
+    "labeling_thought": "Elrontottam a ZH-t/TZ-t, biztos rossz tanuló vagyok.",
+    "magnification_of_the_negative": "A Negatívumok felerősítése",
+    "magnification_of_the_negative_explanation": "Ha figyelmen kívül hagyod a sikereket, akkor lehet, hogy \"rontóc\" vagy. Hogyha folyton az elbaltázott interjúkon rágódsz, akkor lehet, hogy nem jut eszedbe, hogy mennyi új tudásra tettél szert általuk.",
+    "magnification_of_the_negative_thought": "Figyeltem a kajálásra a héten, de kihagytam a futást.",
+    "mind_reading": "Gondolatolvasás",
+    "mind_reading_explanation": "Ha azon rágódunk, hogy mások mit gondolnak rólunk, akkor valószínűleg gondolatot szeretnénk olvasni. Ameddig nem mondják meg, hogy mit gondolnak rólunk, addig nem tudhatjuk, hogy tényleg az-e a helyzet, amit mi feltételeztünk. Miért is kéne mindig a legrosszabbra felkészülni?",
+    "mind_reading_thought": "Bunkó voltam Gyurival, mert nem mentem el a bulijára, most biztos utál engem.",
+    "minimization_of_the_positive": "A Sikerek elhallgatása",
+    "minimization_of_the_positive_explanation": "Ha lebecsüljük a sikereinket, akkor minimalizáljuk a (dopamint-tüzelést) [ez hogy mondjuk magyarul? ¿Öröm?]. Akkor is, ha az első napunk nem ment 100%-ig a tervnek megfelelően, nem jelenti azt, hogy pl. 60% ami jól ment azt ignorálhatjuk.",
+    "minimization_of_the_positive_thought": "Sok embernek tetszett az előadásom, de elbaltáztam a bevezetést, szóval biztos elcsettintettem.",
```

Comment:
... szóval mégiscsak borzasztó volt az egész. (so that, instead of repeating "I messed it up", we also include the conclusion of "... and thus the entire thing is doomed")